src/langcheck/metrics/eval_clients/__init__.py

```diff
@@ -40,3 +40,13 @@
 else:
     __all__.append("PrometheusEvalClient")
     __all__.append("LlamaEvalClient")
+
+try:
```

Comment:
IIRC, this `try...except` block is to avoid exception when the modules with some optional dependencies (`google-generativeai`.. etc) are being loaded.

The only external dependency of `OpenRouterEvalClient` seems to be `requests`. Because `transformers` requires it, `requests` is already one of our implicit requirements. So I think it is okay to

- Write `requests` as our dependency in `pyproject.toml` explicitly
- Directly add `"OpenRouterEvalClient"` to the initialization of `__all__`.